plugin/api.go

```diff
@@ -282,6 +282,11 @@ type API interface {
 	// Minimum server version: 5.2
 	GetChannel(channelId string) (*model.Channel, *model.AppError)
 
+	// GetChannels gets channels by options
+	//
+	// Minimum server version: 5.16
```

Comment:
I was wondering how this sem version was determined?

Reply:
You can just set it to 5.18, which will be the next feature release.